mcs/class/System.Runtime.Serialization.Formatters.Soap/System.Runtime.Serialization.Formatters.Soap/System.Runtime.Serialization.Formatters.Soap.csproj

```diff
@@ -14,5 +14,10 @@
   <ItemGroup>
     <Compile Include="..\..\..\build\common\MonoTODOAttribute.cs" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.1.1" />
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.5.103" />

```

Comment:
These versions should be updated in the Directory.Build.props/targets instead.

Reply:
done